charts/merlin/README.md

```diff
@@ -54,8 +54,14 @@ $ helm uninstall merlin
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 This includes the dependencies that were installed by the chart. Note that, any PVCs created by the chart will have to be deleted manually.
 
-## Configuration
+### Rendered field
+* The purpose of `.Values.rendered.releasedVersion` is to update all configs that share this tag using this 1 value in values.yaml
+`.Values.rendered.releasedVersion` is used in rendering `merlin.config` partial template and `merlin.deploymentTag` partial template
+If .Values.deployment.image.tag is specified, it will overwrite the value in .Values.releasedVersion
+Values.  Rendered values will overwrite values in `.Values.config`
```

Comment:
Is `Values.` supposed to be removed here? 🤔

Reply:
I was also wondering why it was written as such..it turns out my spacebar has issues. Will fix all the places with this issue 😅